pantheon/src/main/java/tech/pegasys/pantheon/cli/error/PantheonExceptionHandler.java

```diff
@@ -32,7 +32,7 @@ public PantheonExceptionHandler(final Supplier<Level> levelSupplier) {
   public List<Object> handleParseException(
       final CommandLine.ParameterException ex, final String[] args) {
     final Level logLevel = levelSupplier.get();
-    if (logLevel != null && Level.DEBUG.isMoreSpecificThan(logLevel)) {
+    if (ex.getMessage() == null || (logLevel != null && Level.DEBUG.isMoreSpecificThan(logLevel))) {
```

Comment:
Just checking you meant to include this change?

Reply:
it was a debugging aid that I forgot to clean up, so I'll cut it.  May be worthy of its own PR.